hedyorg/hedy
# [FIX] Fix errors not being displayed properly

**Description**

After we merged #4341 there was a error were the errors were not being shown properly. This was a front-end error that happened because I didn't change a coupe of calls to the markers interface. This PR fixes that, in a way that needs to be addressed furthermore when we keep on working on #4382, namely: the Hedy Interface shouldn't have Markers inside it.

**Fixes #4393 **

**How to test**

Make some errors and see if the skipping errors feature works properly and the errors are being shown.

**Checklist**
Done? Check if you have it all in place using this list:*
  

If you're unsure about any of these, don't hesitate to ask. We're here to help!
